sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -1156,16 +1217,13 @@
   }
  ],
  "metadata": {
-  "interpreter": {
-   "hash": "da404a94b19d2e6a57be28cbcf6e71fbd41612916c3423bc5e257c11b3d83fa0"
-  },
   "kernel_info": {
    "name": "python3-azureml"
   },
   "kernelspec": {
```

Comment:
I don't think we should commit the kernelspec changes

Reply:
That's a good point. I will check how to remove these. If you know, can you please share the information?